test/run.d

```diff
@@ -145,7 +181,29 @@ auto predefinedTargets(string[] targets)
         return testPath(dir).dirEntries("*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
     }
 
-    Appender!(string[]) newTargets;
+    static Target createUnitTestTarget()
+    {
+        Target target = {
+            args: [getEnvironment["DMD"], "-run", "unit_test_runner.d"]
+        };
+
+        return target;
+    }
+
+    static Target createDefaultTarget(string filename)
+    {
+        Target target = {
+            filename: filename,
+            args: [
+                resultsDir.buildPath("d_do_test"),
```

Comment:
Not that we did this in the past, but if we already change this, let's better make an `enum d_do_test = ...`

Reply:
`enum d_do_test = "d_do_test"` or what do you refer to? I would not consider that a good name for the constant.

Reply:
Well my point was that this value is a constant and could be defined as such statically.